bip-taro-proof-file.mediawiki

```diff
@@ -58,48 +58,77 @@ its the genesis outpoint.
 
 ====File Serialization====
 
-A single inclusion and state transition proof has the following format:
-* <code>previous_outpoint || block_header || merkle_inclusion_proof || anchor_transaction || tlv_proof_map</code>
+A single inclusion and state transition proof has the following format is a TLV
+blob with the following format:
+* type: 0 (<code>prev_out</code>)
+** value: 
+*** [<code>36*byte</code>:<code>txid || output_index</code>]
+* type: 1 (<code>block_header</code>)
+** value: 
+*** [<code>80*byte</code>:<code>bitcoin_header</code>]
+* type: 2 (<code>anchor_tx</code>)
+** value: 
+*** [<code>...*byte</code>:<code>serialized_bitcoin_tx</code>]
+* type: 3 (<code>anchor_tx_merkle_proof</code>)
```

Comment:
Aren't these basically SPV proofs? Wondering if we should re-use that vocabulary here.

Reply:
Yeah, but IMO SPV is more in the context of mobile/light wallets. Ultimately, it's a merkle proof triggered by a bloom filter match (which may contain several txns in one), but in our case we just care about proving a single transaction.